com.eduke32.EDuke32.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: com.eduke32.EDuke32
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: eduke32
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # For storing game files
+  - --filesystem=xdg-documents/EDuke32:create
+  # Unfortunately doesn't seem to respect XDG
+  - --persist=.config/eduke32
+add-extensions:
+  com.eduke32.EDuke32.Plugin:
+    versions: 1-3.32
+    directory: extensions
+    subdirectories: false
```

Comment:
Following previous discussion: https://github.com/flathub/flathub/pull/600#discussion_r311439220
`subdirectories: true` is for multiple extensions under the same point, `subdirectories: false` is for single extension.
For the later, extension point should match exact extension ID, e.g. `com.eduke32.EDuke32.DukeNukem3DShareware` both here and for the extension.

Reply:
I think I understand how it all works now :) Thanks for helping out.